spec/factories/intakes.rb

```diff
@@ -552,7 +552,7 @@ def self.roman_numerals(n)
 
   factory :intake, class: Intake::GyrIntake do
     had_wages { :unfilled }
-    client
+    client { create :client, consented_to_service_at: nil }
```

Comment:
the thought process behind this (to the limited extent there was one) is that we start to treat someone as a "client" in the hub once they consent to service -- they are not viewable until then. So most of the hub stuff relies on the concept of a consented client. Alternatively, in the intake we generally start with an unconsented client and they grow into being consented.

Realistically, though, this solves a problem of all of the created clients as a result of creating an intake and a client at the same time from having TWO consented clients.

With the way our factories are set up, 

`create :client, intake: (create :intake)` does a create on the intake, which creates a client and then creates ANOTHER client. Realistically, it should be a `create :client, intake: (build :intake)` to prevent that behavior, but it's so widespread (and in the case of a similar problem on tax returns, prevents transition of tax returns to other states 😬 ) that this creates a bit of a work around.